app/views/student/form.php

```diff
@@ -35,4 +614,66 @@
       <input type="submit" name="edit" value="Update Profile" />
     </form>
   </div>
-</panel>
\ No newline at end of file
+</panel>
+
+<script type="text/javascript">
+function educationValueChanged() {
+  if($('.undergraduate').is(":checked"))
+  {
+    $("#uDegreeChooser").prop('required', true);
```

Comment:
Instead of having each element set required to true individually, have a class that groups these elements together and only one call to .prop('required', true)